components/README.md

```diff
@@ -37,7 +37,6 @@ can be found [here](https://github.com/opendatahub-io/opendatahub-operator/tree/
       GetManagementState() operatorv1.ManagementState
       SetImageParamsMap(imageMap map[string]string) map[string]string
       UpdatePrometheusConfig(cli client.Client, enable bool, component string) error
-      WaitForDeploymentAvailable(ctx context.Context, r *rest.Config, c string, n string, i int, t int) error
```

Comment:
I would challenge the usefulness of keeping this code snippet if it's not automatically sourced. It's been already outdated.

Reply:
Does it mean it will be automated?

Reply:
i think so :D